Assets/_Scripts/AIScripts/CatchPlayer.cs

```diff
@@ -153,7 +156,7 @@ private bool collidingY()
 
         if(Player.IsHiding && _moveTo.chasing)
         {
-            if (Mathf.Abs(gameObject.transform.position.y - playr.transform.position.y) <= 10)
+            if (Mathf.Abs(gameObject.transform.position.y - playr.transform.position.y) <= 5)
```

Comment:
I hate when yall don't make a inspector variable and hard code this stuff 😂 . Forces yall to change the hard code and I hate it